profiles/standard/config/layout.layout.home.json

```diff
@@ -0,0 +1,184 @@
+{
+    "_config_name": "layout.layout.home",
+    "path": "home",
+    "name": "home",
+    "title": "Home page",
+    "description": null,
+    "module": null,
```

Comment:
If we set `module` = `standard` it will in theory enable the ability to revert/disable this layout instead of deleting it.